R/provider-cortex.R

```diff
@@ -20,6 +20,16 @@ NULL
 #' previous messages. Nor does it support registering tools, and attempting to
 #' do so will result in an error.
 #'
+#' `chat_cortex()` picks up the following ambient Snowflake credentials:
```

Comment:
I wonder if it's worth having a standard `## Auth` header?

Reply:
Happy to do that.